2025-03-27 13:23:37,109 INFO    Thread-6 (_run_job):73934 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-27 13:23:37,109 INFO    Thread-6 (_run_job):73934 [wandb_setup.py:_flush():67] Configure stats pid to 73934
2025-03-27 13:23:37,109 INFO    Thread-6 (_run_job):73934 [wandb_setup.py:_flush():67] Loading settings from /home/gametheorylab/.config/wandb/settings
2025-03-27 13:23:37,109 INFO    Thread-6 (_run_job):73934 [wandb_setup.py:_flush():67] Loading settings from /home/gametheorylab/Desktop/RL-DA6400-assignment_1/cartpole-v1/wandb/settings
2025-03-27 13:23:37,109 INFO    Thread-6 (_run_job):73934 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-27 13:23:37,110 INFO    Thread-6 (_run_job):73934 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/gametheorylab/Desktop/RL-DA6400-assignment_1/cartpole-v1/wandb/run-20250327_132337-q7llewqu/logs/debug.log
2025-03-27 13:23:37,110 INFO    Thread-6 (_run_job):73934 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/gametheorylab/Desktop/RL-DA6400-assignment_1/cartpole-v1/wandb/run-20250327_132337-q7llewqu/logs/debug-internal.log
2025-03-27 13:23:37,110 INFO    Thread-6 (_run_job):73934 [wandb_init.py:init():761] calling init triggers
2025-03-27 13:23:37,110 INFO    Thread-6 (_run_job):73934 [wandb_init.py:init():766] wandb.init called with sweep_config: {'alpha': 0.2900151642689289, 'temperature': 1.3081583573109956}
config: {'_wandb': {}}
2025-03-27 13:23:37,110 INFO    Thread-6 (_run_job):73934 [wandb_init.py:init():784] starting backend
2025-03-27 13:23:37,110 INFO    Thread-6 (_run_job):73934 [wandb_init.py:init():788] sending inform_init request
2025-03-27 13:23:37,112 INFO    Thread-6 (_run_job):73934 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-27 13:23:37,113 INFO    Thread-6 (_run_job):73934 [wandb_init.py:init():798] backend started and connected
2025-03-27 13:23:37,114 INFO    Thread-6 (_run_job):73934 [wandb_run.py:_config_callback():1261] config_cb None None {'alpha': 0.2900151642689289, 'temperature': 1.3081583573109956}
2025-03-27 13:23:37,115 INFO    Thread-6 (_run_job):73934 [wandb_init.py:init():891] updated telemetry
2025-03-27 13:23:37,127 INFO    Thread-6 (_run_job):73934 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-27 13:23:37,609 INFO    MsgRouterThr:73934 [mailbox.py:close():129] Closing mailbox, abandoning 1 handles.
2025-03-27 13:23:37,609 ERROR   Thread-6 (_run_job):73934 [wandb_init.py:init():1478] error in wandb.init()
Traceback (most recent call last):
  File "/home/gametheorylab/miniconda3/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1468, in init
    return wi.init(run_settings, run_config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gametheorylab/miniconda3/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 941, in init
    result = wait_with_progress(
             ^^^^^^^^^^^^^^^^^^^
  File "/home/gametheorylab/miniconda3/lib/python3.12/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 24, in wait_with_progress
    return wait_all_with_progress(
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gametheorylab/miniconda3/lib/python3.12/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 70, in wait_all_with_progress
    return _wait_handles(handle_list, timeout=progress_after)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gametheorylab/miniconda3/lib/python3.12/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 110, in _wait_handles
    results.append(handle.wait_or(timeout=remaining_timeout))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gametheorylab/miniconda3/lib/python3.12/site-packages/wandb/sdk/mailbox/mailbox_handle.py", line 122, in wait_or
    return self._fn(self._handle.wait_or(timeout=timeout))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gametheorylab/miniconda3/lib/python3.12/site-packages/wandb/sdk/mailbox/response_handle.py", line 98, in wait_or
    raise HandleAbandonedError()
wandb.sdk.mailbox.mailbox_handle.HandleAbandonedError
